core/src/test/scala/unit/kafka/log/LogCleanerIntegrationTest.scala

```diff
@@ -46,7 +44,7 @@ class LogCleanerIntegrationTest extends AbstractLogCleanerIntegrationTest with K
     TestUtils.clearYammerMetrics()
   }
 
-  @Timeout(value = DEFAULT_MAX_WAIT_MS, unit = TimeUnit.MILLISECONDS)
+  @Timeout(90)
```

Comment:
Since we use mock time, why do we need a much longer timeout?

Reply:
you're right.  Let me remove the Timeout.